.changeset/gold-tips-deliver.md

```diff
@@ -0,0 +1,9 @@
+---
+"@kaizen/notification": major
```

Comment:
Should include @kaizen/rich-text-editor (which depends on @kaizen/notification) too, since it will also require React 18 with this change.